src/taa/inp.100

```diff
@@ -0,0 +1,747 @@
+; Reconstructed 2018 from SYS2; TS MSEND dated 1984-01-14, which is
+; a PDUMP binary assembled from DM: SYSENG; MSEND 140 on 1980-07-28.
+
+SUBTTL READER
+
+DEFINE BUFMAKE NAME,LEN,HLP,TXT
+ZZ==LEN/5
+ZZZ==.
+	BLOCK ZZ
+NAME:	[ASCIZ /TXT/]
+	440700,,ZZZ
+	440700,,ZZZ
+	440700,,ZZZ
+	5*ZZ
+	5*ZZ
+	0
+	0
+	HLP
+TERMIN
+
+.BEGIN READER
+
+CBUF=16
+FINBLN==240
+
+BUFMAKE FILBUF,175,HLPFIL,[File:  ]
+
+FINBUF:	BLOCK 40
+
+FILBLK:
+FILDEV:	SIXBIT /DSK/
+FILSNM:	0
+FILFN1:	0
+FILFN2:	SIXBIT />/
+
+DSKI:		0
+ENDFIL:		0
+PRMPT2:		0
+.U"INREAD:	0
+.U"RQUOTE:	0
+.U"XCTRUB:	0
+.U"TOFCI:	0
+.U"TOERS:	0
+CHPOS:		0
+CVPOS:		0
+.U"KEEPQT:	0
+.U"CRBRK:	0
+CTRLYQ:		-1
+.U"LSTBRK:	0
+
+.U"BUFPRM==0
+.U"BUFBEG==1
+.U"BUFFCH==2
+.U"BUFNCH==3
+.U"BUFLEN==4
+.U"BUFVLN==5
+.U"BUFCCT==6
+.U"BUFVCT==7
+.U"BUFHLP==10
+
+RCMDNX:	MOVEM	B,BUFNCH(CBUF)
+	SKIPGE	C
+	 MOVEI	3,0
+	MOVEM	C,BUFVCT(CBUF)
+	JRST	RCMD
+
+.U"GETBUF:
+	PUSH	P,CBUF
+	MOVE	CBUF,B
+	PUSH	P,C
+	PUSH	P,D
+RCMDXX:
+RCMD:	MOVE	B,BUFNCH(CBUF)
+	MOVE	C,BUFVCT(CBUF)
+	SKIPE	TOERS
+	 JUMPN	C,[PUSHJ P,REPPER ? JRST RCMD0]
+	PUSHJ	P,PPRMPT
+RCMD0:	SETOM	INREAD
+RCMD1:	.IOT	TTYI,A
+	SKIPE	RQUOTE
+	 JRST	[SETZM	RQUOTE
+		 SKIPE	KEEPQT
+		  JRST	RCMDL
+		 SKIPN	TOERS
+		  JRST	RCMDL
+		 PUSH	P,A
+		 MOVEI	A,21
```

Comment:
Maybe `^Q`?  I haven't checked what the code does, but RQUOTE seems to indiate something with quoting.

Reply:
It feeds it into CHRTYP, so yes, ^Q.